linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -238,28 +238,56 @@ if (env.performance == "yes") {
+        echo "Send email with kernel validation results link"
+        if (env.kernel == "linux-azure") {
+            version_identifier = "_azure"
+        } else {
+            version_identifier = "_edge"
+        }
+        kernels_info = readFile (LATEST_VERSION_LOCATION)
+        kernels_info = kernels_info.split("\n")
+        kernels_info.each {
+            if (it.contains("${env.distro}${version_identifier}")) {
+                latest_kernel_version = it.split("=")[1]
+            }
+        }
         stage('Send-email') {
-            withCredentials([string(credentialsId: 'MAIL_LIST', variable: 'MAIL_LIST')]) {
-                echo "Send email with kernel validation results link"
-                if (env.kernel == "linux-azure") {
-                    version_identifier = "_azure"
-                } else {
-                    version_identifier = "_edge"
-                }
-                kernels_info = readFile (LATEST_VERSION_LOCATION)
-                kernels_info = kernels_info.split("\n")
-                kernels_info.each {
-                    if (it.contains("${env.distro}${version_identifier}")) {
-                        latest_kernel_version = it.split("=")[1]
-                    }
-                }
+            withCredentials([string(credentialsId: 'MAIL_LIST_OWNER', variable: 'MAIL_LIST_OWNER')]) {
                 emailext (
                     subject: "Test results for ${env.distro} ${latest_kernel_version} ${env.kernel} kernel",
-                    to: "${env.MAIL_LIST}",
+                    to: "${env.MAIL_LIST_OWNER}",
                     mimeType : "text/html",
                     body: '${SCRIPT, template="ubuntu.template"}'
                 )
             }
         }
+        stage('Kernel-SignOff') {
+            withCredentials([string(credentialsId: 'MAIL_LIST_SIGNOFF', variable: 'MAIL_LIST_SIGNOFF')]) {
+                try {
+                    checkout scm
+                    echo "Ask for manual sign-off from the owner"
+                    timeout (time: 72, unit: 'HOURS') {
+                        input(message: "Are you sure you want to sign off ${env.distro} ${latest_kernel_version} ${env.kernel} kernel?")
+                    }
+                    emailext (
+                        subject: "${env.distro} ${latest_kernel_version} ${env.kernel} is OK and can be signed off!",
+                        to:"${env.MAIL_LIST_SIGNOFF}",
+                        mimeType : "text/html",
+                        body: """
+                            Hello,<br/><br/>
+                            ${latest_kernel_version} ${env.kernel} kernel for ${env.distro} is OK and can be signed off!<br/><br/>
+                            Thank you,<br/>Jenkins CI
+                        """
+                    )
+                } catch (exc) {
+                    emailext (
+                        subject: "${latest_kernel_version} ${env.kernel} kernel for ${env.distro} can't be signed off!",
```

Comment:
ditto ! mark

ditto who did not sign off,

Reply:
will change